src/Hive.IO/Hive.IO/GhInputOutput/GhResults.cs

```diff
@@ -212,7 +212,16 @@ protected override void SolveInstance(IGH_DataAccess DA)
                 List<GH_Number> numbers = iQsPerWin[path];
                 QsPerWindow[i] = new double[numbers.Count];
                 for (int u = 0; u < numbers.Count; u++)
-                    QsPerWindow[i][u] = (double)numbers[u].Value / 1000.0;
+                {
+                    try
+                    {
+                        QsPerWindow[i][u] = (double)numbers[u].Value / 1000.0;
+                    }
+                    catch
+                    {
+                        QsPerWindow[i][u] = 0.0;
+                    }
```

Comment:
this is not very elegant, but I couldn't figure out another way to make it work @daren-thomas 

`if(!( (double)numbers[u].Values is null) )` wouldn't work because `.Values` won't exist